docs/head.md

```diff
@@ -57,6 +57,24 @@ export default {
 </script>
 ```
 
+If you wish to access `this` or data on a component/page, make `metaInfo` a function:
+```js
+<script>
+export default {
+  name: 'Post Template',
```

Comment:
Is this necessary? I never assign my templates a name.

Reply:
Check out https://gridsome.org/docs/components/

Reply:
It is unnecessary but recommended.

Reply:
Jumping in here - My understanding is that adding a `name` is not a best practice so much as a convenience for debugging. [This SO post](https://stackoverflow.com/questions/50457643/vue-js-single-file-component-name-not-honored-in-consumer) calls this out since the component name doesn't affect what you import a component as later.

That said, I think it makes sense to include the `name` to correspond to what users may expect. I would place templates at the same hierarchy as the `App.vue` file that gets created when users run `vue create`, and `App.vue` includes `name: "App"` by default.

Reply:
Thanks for the additional context @12vanblart! That sounds reasonable.